docs/source/release_notes/release-notes.rst

```diff
@@ -52,9 +52,9 @@ develop
 
     *    - |devbreak|
          - The ``putUnlessExists`` API has been removed from AtlasDB tables, as it was misleading (it only did the put if the given row, column and value triple were already present, as opposed to the more intuitive condition of the row and column value pair being present).
-           Please replace any uses of ``putUnlessExists`` with a get, check and put - these will still be transactional because of the AtlasDB transaction protocol.
+           Please replace any uses of the table-level ``putUnlessExists`` with a get, check and put if appropriate - these will still be transactional because of the AtlasDB transaction protocol.
            Note that this is not the same as the KVS ``putUnlessExists`` API, which is still used by the transaction protocol.
-           This API has already been deprecated for 11 months.
+           This API has already been deprecated for 10 months (since August 2017).
```

Comment:
nit: the relative value will be wrong next month

Reply:
I'll put the exact date first. I've changed the relative part to say `11 months from time of writing` (how long it has been in this state for is also important!)